Gulpfile.js

```diff
@@ -0,0 +1,52 @@
+var gulp = require('gulp');
+var uncss = require('gulp-uncss');
+
+gulp.task('uncss', function() {
+  return gulp.src([
+      '_site/lib/material-design-lite/material.min.css',
+      '_site/lib/foundation/css/foundation.css'
+    ])
+    .pipe(uncss({
+      html: [
+        "http://localhost:4000/404.html",
```

Comment:
Using https://www.npmjs.com/package/gulp-uncss-sp we could glob our way to all generate html files in _site. Otherwise we could script it by building an object on the fly, will do.